src/client/containers/ProfilePage/ProfilePage.styles.css

```diff
@@ -91,8 +91,48 @@ div.frame {
 }
 
 .profile-career-container .profile-career-title {
-  font-size: 20px;
+  font-size: 1.5625rem;
   font-weight: bold;
   margin: 10px 0 40px 10px;
   line-height: 10px;
```

Comment:
Could you update the line-heights given in pixels to rem? because at times the line-height can be quite small for the font size, so opt for using the same units for all font related properties

Reply:
ok @valmdz

Reply:
Hi @soujanyapolavarapu  My comment regarding the flex-direction is more a suggestion, not necessarily something you need to implement. It is more so you are aware that on smaller screens the page is not very responsive.

Reply:
ok @valmdz Thank you :)